moderation/nickname-moderation.go.tmpl

```diff
@@ -0,0 +1,62 @@
+{{/*
+	Custom Command for nickname moderation
+	If a user of your server has an inappropriate or unpingable nickname, mods often have to change their nickname manually.
+	With this custom command, they can moderate nicknames with a simple command.
+	After you moderated the nickname, the nickname will be changed when the moderated user sends their next message.
+	The command nick-numberreset makes the next moderated nickname start at number one.
+
+	Trigger type: Regex
+	Trigger: \A
+
+	Usage:
+	-modnick <@user/ID>
+	-unmodnick <@user/ID>
+	-nick-numberreset
+
+	Credit: 2x2Master#7363
+	Feel free to ping me in YAGPDB support if you have any questions or problems!
+*/}}
+
+{{/* Configuration area starts */}}
+{{$prefix := "-"}}{{/* YAGPDB's prefix in your server */}}
+{{$name := "Moderated Nickname"}}{{/* What should be in front of the numbers when a nickname is moderated */}}
+{{$padding := "5"}}{{/* How many digits the number should have, recommendation: between 3 and 7 */}}
+{{$modRoles := cslice 809781706658807838 815197447000555530}}{{/* Users with these roles can moderate nicknames */}}
+{{/* Configuration area ends */}}
+
+
+{{$mod := false}}
+{{range $modRoles -}}
+	{{if hasRoleID .}}
+		{{$mod = true}}
+	{{end}}
+{{end}}
+{{if eq (lower (index .CmdArgs 0)) (print $prefix "modnick")}}
+	{{if $mod}}
+	{{((userArg (index .CmdArgs 1)).String)}}'s nickname will be moderated when they next send a message.
```

Comment:
You are never checking for an existence of (index .CmdArgs 1). You can do this like so:
```{{if ge (len .CmdArgs) 2}}```. Same for all commands.